packages/registry/src/addon-registry.js

```diff
@@ -218,6 +237,13 @@ class AddonConfigurationRegistry {
   }
 
   initPublishedPackage(name) {
+    // I am in the paths list, if so, register it as a development package
+    // instead than a published one
+    const [, pathsConfig] = this.getTSConfigPaths();
+    if (pathsConfig.hasOwnProperty(name)) {
+      this.initDevelopmentPackage(name);
```

Comment:
I think you can `return` here? The code under this line will not execute because of the `if`, anyway